### PR TITLE
Uploading the seventeen point: iframe

### PR DESCRIPTION
**Pull request by juanse962**
Delivering W3school iframes activity

### CODE REVIEW

File: cypress/support/commands.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
 import "cypress-file-upload";
+import "cypress-iframe";
+require("cypress-iframe");
```
Comment (holgiosalos) [inline cypress/support/commands.js]: just one of these two is required, please delete one

Comment (holgiosalos) [inline cypress/support/commands.js]: just one of these two is required, please delete one

**Review by holgiosalos — APPROVED** [review]
Great job!

**Comment by juanse962** [comment]
Thanks for the knowledge provided

**Pull request merged**